src/test/java/uk/gov/hmcts/reform/civil/handler/callback/user/EvidenceUploadHandlerTest.java

```diff
@@ -45,24 +62,148 @@ void givenAboutToStartThenReturnsAboutToStartOrSubmitCallbackResponse() {
         assertThat(response).isInstanceOf(AboutToStartOrSubmitCallbackResponse.class);
     }
 
+    @Nested
+    class MidEventValidateValuesCallback {
+        private static final String PAGE_ID = "validateValues";
+
+        @Test
+        void shouldReturnError_whenExpertOption1UploadDateFuture() {
+            List<Element<UploadEvidenceExpert>> date = newArrayList();
+            date.add(0, element(uploadEvidenceDate.toBuilder()
+                                                .expertOption1UploadDate(LocalDate.now().plusWeeks(1)).build()));
```

Comment:
After you replaced the occurrence of `LocalDate.now()` in `EvidenceUploadHandler` with the Time bean, create a mock here and use it to test. I can see that you are testing the error path here but not the happy path on the individual dates: the change in `EvidenceUploadHandler` and in the test will enable you to test also the happy path for all the dates.